### PR TITLE
import paddle by default

### PR DESCRIPTION
**Pull request by TomorrowIsAnOtherDay**

### CODE REVIEW

File: benchmark/fluid/A2C/README.md [hunk 1]
```diff
@@ -14,15 +19,15 @@ Performance of A2C on some envrionments in training process after 10 million sam
 
 ## How to use
 ### Dependencies
-+ [paddlepaddle>=2.0.0](https://github.com/PaddlePaddle/Paddle)
-+ [parl>=1.4.3](https://github.com/PaddlePaddle/PARL)
++ [paddlepaddle>=1.8.5](https://github.com/PaddlePaddle/Paddle)
```
Comment (zenghsh3) [inline benchmark/fluid/A2C/README.md]: For the examples of fluid, the requirement of paddlepaddle should be <2.0.0?

**Review by zenghsh3 — CHANGES_REQUESTED** [review]

**Comment by zenghsh3** [comment]
The links of examples in [Chinese document](https://github.com/PaddlePaddle/PARL/blob/zhoubo01/docs/zh_CN/Overview.md) are pointing to the fluid version (hardcoded with `CN_docs` branch).

File: benchmark/fluid/A2C/README.md (hunk 1 above)
Comment (TomorrowIsAnOtherDay) [reply benchmark/fluid/A2C/README.md]: fixed

**Review by zenghsh3 — APPROVED** [review]
LGTM

**Pull request merged**